lib/matplotlib/widgets.py

```diff
@@ -702,14 +702,18 @@ def __init__(
             f'marker{k}': v for k, v in {**defaults, **handle_style}.items()
         }
 
+        verts = np.zeros([5, 2])
```

Comment:
You can also do
```suggestion
        verts = self._fill_verts(np.zeros([5,2]), valinit)
```

and then eliminate the usage in the if